src/panels/config/zha/zha-devices-data-table.ts

```diff
@@ -101,6 +107,14 @@ export class ZHADevicesDataTable extends LitElement {
       .getAttribute("data-row-id")!;
     showZHADeviceInfoDialog(this, { ieee });
   }
+
+  static get styles(): CSSResult {
+    return css`
+      ha-data-table {
+        height: 600px;
```

Comment:
I wonder if we can add an option for the data table to just show all items ? As that's how ZHA is currently using it, no ?

Reply:
Which means that height = rowCount * rowHeight + headerHeight

Reply:
Without the virtualization?